app/views/coronavirus_form/business_details.html.erb

```diff
@@ -26,7 +26,7 @@
     text: t('coronavirus_form.questions.business_details.company_name.label'),
   },
   error_message: error_items('company_name'),
-  value: session.dig("business_details", "company_name"),
+  value: @form_responses.fetch(:$1, :$2),
```

Comment:
To be honest I think it's easier to read if the explicit values are kept in. `dig` can be used with symbols, so `@form_responses.dig(: business_details, : company_name)` would work. This applies to the other views too.

Reply:
🤦‍♂ This is an epic find and replace fail